monolithe/generators/lang/go/templates/model.go.tpl

```diff
@@ -15,9 +15,18 @@ var {{specification.entity_name}}Identity = bambou.Identity {
 type {{specification.entity_name_plural}}List []*{{specification.entity_name}}
 
 // {{specification.entity_name_plural}}Ancestor is the interface of an ancestor of a {{specification.entity_name}} must implement.
+// An Ancestor is defined as an entity that has {{specification.entity_name}} as a descendant.
```

Comment:
"An Ancestor is defined an entity that has ... as a descendant" 
or
"An Ancestor is defined as an entity that has ... a descendant"

Both sound very strange